tests/test_bundle.py

```diff
@@ -265,6 +265,26 @@ def _test_bundle_put(self, replica: Replica, fixtures_bucket: str):
                 expected_code=requests.codes.bad_request
             )
 
+        with self.subTest(f'{replica}: should *NOT* be able to put a bundle containing a duplicate filename'):
```

Comment:
I prefer we use the "`result` when `condition`" format when describing test when possible. I've been changing them whenever I have to modify any tests. I suggest you change
> "should *NOT* be able to put a bundle containing a duplicate filename" 

to

> "put fails when the bundle contains a duplicated file name"

Reply:
changed.